azure-cli-extensions.pyproj

```diff
@@ -2468,120 +2468,26 @@
     <Compile Include="src\find\setup.py" />
     <Compile Include="src\front-door\azext_front_door\commands.py" />
     <Compile Include="src\front-door\azext_front_door\custom.py" />
-    <Compile Include="src\front-door\azext_front_door\vendored_sdks\front_door_management_client.py" />
```

Comment:
could we remove this file?

Reply:
Yes, the new vendored SDK directory structure has the same file as '_front_door_management_client.py'

---
In reply to: [398370729](https://github.com/Azure/azure-cli-extensions/pull/1432#discussion_r398370729) [](ancestors = 398370729)